### DISCUSSION TITLE
[2.24.0][PG] Unknown arg `path`or type JsonFilter

### DISCUSSION THREAD

**Discussion by faustbrian** [prisma/prisma#7407] [Q&A]
Using 2.24.0 with `previewFeatures = ["filterJson"]` and according to the documentation over at https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-json-fields something like this should work.

```ts
const getUsers = await prisma.user.findMany({
  where: {
    extendedPetsData: {
      path: ['petName'],
      equals: 'Claudine',
    },
  },
})
```

But executing this example results in the same error that I'm experiencing with my own code.

```ts
PrismaClientValidationError:
Invalid `prisma.signal.findMany()` invocation:

{
  where: {
    category: {
      path: [
      ~~~~
        'description'
      ],
      string_contains: 'Technical'
      ~~~~~~~~~~~~~~~
    }
  },
  orderBy: {
    date_created: 'desc'
  },
  skip: 0,
  take: 15
}

Unknown arg `path` in where.category.path for type JsonFilter. Did you mean `not`? Available args:
type JsonFilter {
  equals?: Json
  not?: Json
}
Unknown arg `string_contains` in where.category.string_contains for type JsonFilter. Available args:

type JsonFilter {
  equals?: Json
  not?: Json
}
```

Is there some syntax error or misconfiguration that I'm overlooking that causes this error?

**Comment by ryands17** [comment]
@faustbrian 👋 

Did you run `prisma generate` after adding the preview feature?

**Reply by faustbrian** [reply]
It automatically runs before each build but I also tried to manually run it and still encounter this error.

**Reply by ryands17** [reply]
It would be great if you could provide a minimal reproduction so that I can check.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
